### ISSUE TITLE
RegisterJobNameVisibilityCheckerPass class has been removed

### ISSUE THREAD

**Issue by gplanchat** [akeneo/pim-docs#553]
In file [AcmeNotifyConnectorBundle.php](src/Acme/Bundle/NotifyConnectorBundle/AcmeNotifyConnectorBundle.php) there is a mention of class `Pim\Bundle\ImportExportBundle\DependencyInjection\Compiler\RegisterJobNameVisibilityCheckerPass`.

This class has been removed since 1.7 (see [CHANGELOG-1.7.md](https://github.com/akeneo/pim-community-dev/blob/master/CHANGELOG-1.7.md#classes-1)).

The doc should be updated.

Thus, there should also be some doc to explain how to build another tab in the job profiles view, I can't find any, or probably I didn't find it.

**Comment by mickaelandrieu** [comment]
Hello,
yes, the class was removed ... I don't know how we should update this part.

What do you think @jjanvier ? I can update the docs but I need more information.

**Comment by gplanchat** [comment]
In fact, I already updated the docs in the mean time in akeneo/pim-docs#563

**Comment by mickaelandrieu** [comment]
@gplanchat yes, but the code in the Bundle is still wrong (as the class doesn't exits. Should we unregister the compiler pass or totally remove the AcmeNotifyConnector Bundle?

**Comment by gplanchat** [comment]
The compiler pass is now useless

**Comment by mickaelandrieu** [comment]
@gplanchat ok, we need to update the bundle?

**Comment by gplanchat** [comment]
Yes, for this part @phaseinducer did update the docs at https://docs.akeneo.com/latest/cookbook/import_export/create-connector.html#configure-the-ui-for-our-new-job

The product export jobs have an additional "content" tab since 1.6. The method to add it in 1.6 was to call the `RegisterJobNameVisibilityCheckerPass` in one's bundle class. Since 1.7 there is another approach, very different than what was made in 1.6 - 1.6 was Symfony-based, the 1.7 version is Javascript-based. The `AcmeAppBundle` should be updated in this way.

**Comment by mickaelandrieu** [comment]
Ok, I'll ask @phaseinducer if he can help to fix the docs then :)

Thanks for the report Greg!
